docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx

```diff
@@ -159,7 +159,7 @@ Also known as **US902-928**
 | Chad                             |                     |                                                                                                                               |
 | Chile                            | AU915-928           | [FIJA NORMA TECNICA DE EQUIPOS DE ALCANCE REDUCIDO](https://www.leychile.cl/Consulta/m/norma_plana?org=&idNorma=240404)       |
 | China                            | CN470-510 CN779-787 |                                                                                                                               |
-| Colombia                         | US902-928           |                                                                                                                               |
+| Colombia                         | AU915-928           | https://lora-alliance.org/resource_hub/rp2-1-0-3-lorawan-regional-parameters/                                                                                                                              |
```

Comment:
```suggestion
| Colombia                         | AU915-928           |                                                                                                                           |
```

Reply:
I wouldn't consider the LoRaWAN regional spec as a "regulatory document". A regulatory document should be a regional regulation IMO, such as FCC in the US.

Otherwise, the change looks good!

Reply:
just updated it with the regulatory entity reference doc. @lthiery